.github/workflows/release.yml

```diff
@@ -3,7 +3,7 @@ name: Tag and release
 on:
   push:
     branches:
-      - main
+      - 4.x
```

Comment:
will this branch be merged into main? in that case I suggest keeping main here

Reply:
Yeah, I hadn't decided whether we would release 4.x off of this and keep main as a 3.x stream, since we'll likely be stuck on that in PE for a while. But I think that doesn't make sense, and we should branch `3.x` when the time comes, then merge this into main. I'll take this out.